src/web/server/document.tsx

```diff
@@ -55,6 +55,74 @@ export const document = ({ data }: Props) => {
 		),
 	);
 
+	const loadableExtractor = new ChunkExtractor({
+		stats: loadableManifestJson,
+		entrypoints: ['react'],
+	});
+
+	// The lodable-components docs want us to use extractor.collectChunks() but
+	// we don't have the traditional same <App /> rendered on server and client.
+	// Our data structure is vastly different (CAPIType vs CAPIBrowserType) and
+	// our architecture expects src/web/App to be client-side only, therefore
+	// it is difficult for us to use collectChunks to automatically extract the
+	// component splits that we want.
+	// However, this does actually suit our architecture as we can use the CAPI
+	// component reference.
+	const allChunks: LoadableComponents[] = [
+		{ chunkName: 'EditionDropdown', alwaysAdd: true },
```

Comment:
> Oh, is it because we always add it so we don't need it?

Yeah, @oliverlloyd, `CAPIElementType` is used for the comparison against the current request's elements. 

What about:

```js
{ chunkName: 'EditionDropdown'},
{
			chunkName: 'elements-YoutubeBlockComponent',
			addWhenElTypeMatches: 'model.dotcomrendering.pageElements.YoutubeBlockElement'
}
```

where *default is always* and conditional is based on the other value?

Reply:
`addWhenElTypeMatches` is a bit verbose, maybe? What about 

```
{
			chunkName: ElementNameType,
			addWhen: ElementType | 'always';
}
```

This is a hard to build...

Reply:
Ah yeah, I prefer the union and required type here